opencog/pln/types/atom_types.script

```diff
@@ -12,16 +12,20 @@ INTENSIONAL_DIFFERENCE_LINK <- ORDERED_LINK,EVALUATABLE_LINK
 
 // For the definition of these links see
 // https://wiki.opencog.org/w/Category:Atom_Types
-// PREDICTIVE_IMPLICATION_SCOPE_LINK <- SCOPE_LINK
 
-// These links are already declared in the spacetime repository, see
-//
-// spacetime/opencog/spacetime/atom-types/atom_types.script.
-//
-// AT_TIME_LINK <- ORDERED_LINK
-// TIME_INTERVAL_LINK <- ORDERED_LINK
-// TIME_NODE <- NODE
-// PREDICTIVE_IMPLICATION_LINK <- ORDERED_LINK
+TIME_NODE <- NUMBER_NODE
+AT_TIME_LINK <- ORDERED_LINK
+TIME_INTERVAL_LINK <- ORDERED_LINK
+
+LATEST_LINK <- ORDERED_LINK
+TIME_DOMAIN_NODE <- NODE
+
+PREDICTIVE_IMPLICATION_LINK <- ORDERED_LINK
+TAIL_PREDICTIVE_IMPLICATION_LINK <- ORDERED_LINK
+
+PREDICTIVE_ATTRACTION_LINK <- ORDERED_LINK
+SIMULTANEOUS_AND_LINK <- ORDERED_LINK //AND_LINK
```

Comment:
In the light of what has been said, @ntoxeg, could you remove these definitions above?